litmus-portal/backend/graphql-server/pkg/database/mongodb/mongodb.go

```diff
@@ -13,95 +13,82 @@ import (
 )
 
 var clusterCollection *mongo.Collection
-var backgroundContext context.Context
+var backgroundContext = context.Background()
```

Comment:
Global context should be reuse by every function

Reply:
It is being reused, the context is being set to context.Background() during the DBInit() call. It is responsible to initialize all DB related data.

Reply:
It is not being reused in this place https://github.com/litmuschaos/litmus/blob/litmus-portal/litmus-portal/backend/graphql-server/pkg/database/db.go#L24

Reply:
Yes that one I had kept as context.TODO, we can change that to use this one, we have to set the variable before that, currently we are initializing it after line 24.